lib/simple-dom/html-serializer.js

```diff
@@ -59,7 +59,8 @@ HTMLSerializer.prototype.escapeText = function(textNodeValue) {
 HTMLSerializer.prototype.text = function(text) {
   var parentNode = text.parentNode;
   if(parentNode && (parentNode.nodeName === "STYLE" ||
-    parentNode.nodeName === "SCRIPT")) {
+    parentNode.nodeName === "SCRIPT" ||
+    parentNode.nodeName === "CODE")) {
```

Comment:
i think this line of code should also be here
https://github.com/canjs/can-simple-dom/blob/master/lib/simple-dom/document/element.js#L169